bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/image/Icon.java

```diff
@@ -66,7 +83,20 @@ public IconType getType() {
      * @return current icon type
      */
     public IconType type() {
-        return iconBehavior.type();
+        return getType();
     }
 
+    @Override
+    public void onComponentTagBody(MarkupStream markupStream, ComponentTag openTag) {
+        super.onComponentTagBody(markupStream, openTag);
+        if(hasIconType() && getType().getTagBody() != null) {
+            replaceComponentTagBody(markupStream, openTag, getType().getTagBody());
+        }
+    }
+
+    @Override
+    protected void onComponentTag(ComponentTag tag) {
+        super.onComponentTag(tag);
+        Components.assertTag(this, tag, "i", "span");
```

Comment:
This might break someone's application.
I know that usually font icons are used with `i` or `span` but I've seen also with `em` and who knows what else could be in use in the wild.
Let's remove this check!

Reply:
In fact this is already done in the IconBehavior, so I can remove it!
So your `em` are already change in `i` with the actual code.